rocketpy/environment/environment.py

```diff
@@ -2965,7 +3025,14 @@ def pressure_function(h):
             outputs="Pressure (Pa)",
         )
 
-        return None
+        # Save international standard atmosphere height by pressure profile
+        # and discretize it. This is done to speed up the calculations in the
+        # trajectory simulation.
```

Comment:
Shorten comments are usually better. Also, comment "why", not "what"

```suggestion
        # Discretize Function to speed up the trajectory simulation.
```